sdk/containerregistry/container-registry/samples/typescript/src/containerRegistryClient.ts

```diff
@@ -5,17 +5,28 @@
  * @summary Demonstrates the use of a ContainerRegistryClient.
  */
 
-import { ContainerRegistryClient } from "@azure/container-registry";
+import { ContainerRegistryClient, ContainerRegistryUserCredential } from "../../../src";
```

Comment:
This was probably helpful while testing, but I think we depend on the package directly and use `dev-tool` to change the imports to use the project package when running build:samples.

Reply:
yeah I know I would love to use what Will demo'd...but the ability to write and debug the same file now made me to do this

Reply:
Fixed